packages/client/components/AddTeamMemberModal.tsx

```diff
@@ -48,13 +87,35 @@ const ErrorMessage = styled(StyledError)({
   marginTop: '.5rem'
 })
 
-const ShareableLink = styled('div')({
+const StyledLabelHeading = styled(LabelHeading)({
+  alignItems: 'center',
   display: 'flex',
-  justifyContent: 'center',
-  // padding (18+8) addresses hanging icon at mobile widths
-  padding: '0 26px'
+  fontSize: 15,
+  lineHeight: '21px',
+  padding: '0 0 16px',
+  textTransform: 'none'
 })
 
+const StyledCopyShortLink = styled(CopyShortLink)({
+  borderRadius: 4,
+  border: `1px dashed ${PALETTE.EMPHASIS_COOL_LIGHTER}`,
+  color: PALETTE.EMPHASIS_COOL,
+  fontSize: 15,
+  margin: '0 0 32px',
+  padding: 11,
+  ':hover': {
+    color: PALETTE.EMPHASIS_COOL_LIGHTER
+  }
+})
+
+const IllustrationBlock = () => {
```

Comment:
@mattkrick I created this component for the hook since the larger component is not a functional component. Okay until we refactor this to a functional component?

Reply:
not sure we need a component, can we just do:
`return showIllustration ? <Illustration/> : null`?

Reply:
Nice!